src/cdk/schematics/utils/build-component.ts

```diff
@@ -40,6 +40,12 @@ import {getProjectFromWorkspace} from './get-project';
 import {getDefaultComponentOptions} from './schematic-options';
 import {ts} from './version-agnostic-typescript';
 
+/**
+ * List of supported styles which are CSS supersets. All supported CLI style extensions can be
+ * found here: angular/angular-cli/master/packages/schematics/angular/ng-new/schema.json#L118-L122
+ */
+const supportedCssSupersets = ['scss', 'less'];
```

Comment:
@jelbourn I decided to do in a safelist-fashion as "technically" the CLI could add new unsupported style extensions which will then break. This way we guarantee that we _never_ generate invalid stylesheets.

Reply:
That's the right call :+1: